public_html/wp-content/mu-plugins/blocks/assets/src/speakers/index.js

```diff
@@ -18,5 +18,9 @@ export const settings = {
 	edit,
 	save        : function() {
 		return null;
+		// should we return a cached copy of the markup here, as a failsafe in case we ever want to disable this block or make some big changes to it?
+		// disabling it entirely seems unlikely, but we might eventually want to build a v2 from scratch or something
+		// not having to worry about back-compat might give us some flexibility in the future
+		// https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-edit-save/#save
```

Comment:
Maybe? I haven't tested this, but my assumption is that the markup would get saved to `post_content` and then when the post is loaded on the front end, we would end up with the cached version, followed by a dynamically generated version. But I could very well be wrong about that.

Reply:
I think that's only true for static blocks; for dynamic blocks, it only saves a comment in `post_content`

```html
<!-- wp:heading -->
<h2>"Heading" is a static block</h2>
<!-- /wp:heading -->

<!-- wp:wordcamp/speakers {"mode":"all","sort":"date_desc","layout":"grid","grid_columns":4,"content":"none","speaker_link":true} /-->
```